src/Utilities/Math.hpp

```diff
@@ -3,7 +3,9 @@
 
 #pragma once
 
+#include <blaze/math/functors/Clamp.h>
```

Comment:
Don't use blaze things directly like this.

Reply:
removed in the last fixup